Assets/4-TNDStudios/GameManager.cs

```diff
@@ -0,0 +1,204 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.Experimental.GraphView;
```

Comment:
Importing `UnityEditor` here will cause an error when building the final game. Can this be removed? Or you can surround all editor code in preprocessor tags (`#if UNITY_EDITOR`).

Reply:
This was added accidentally. I removed it.